### PR TITLE
Examples for episodes 540 - 542

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: sample_code/ep540/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# [I made every sentry page 300ms faster (intermediate)](https://youtu.be/FY7EQwT7QKU)
+
+Today I talk about how I made every sentry.io page ~250-300 ms faster and talk a little bit about the various http caching approaches
+
+## Interactive examples
+
+### Bash
+
+```bash
+curl -I <url>
```
Comment (int3l) [inline sample_code/ep540/README.md:10]: placeholder instead of actual CDN link

Comment (asottile) [reply sample_code/ep540/README.md:10]: the actual link would be fine I think

File: sample_code/ep541/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+echo hi hi >> t
+
+echo hello hello > config/t
+cat !$
+
+babi config/t.tmp
+mv !$ config/t
+
+touch t2
+echo hello hello > t2
```
Comment (int3l) [inline sample_code/ep541/README.md:52]: There is a typo in the video on this line (missing > symbol)

File: sample_code/ep540/README.md (hunk 1 above)
Comment (asottile) [reply sample_code/ep540/README.md:10]: or you can find any one from the sentry login page -- I realize typing one of those out would be a pain in the ass

File: sample_code/ep542/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+./some-exe
+$PWD/some-exe
+
+python
+
+git clone --depth=1 git@github.com:asottile/rename-exchange
+nano rename-exchange/test.sh
+
+git clone --depth=1 git@github.com:pre-commit/pre-commit
+nano pre-commit/pre_commit/resources/hook-tmpl
```
Comment (int3l) [inline sample_code/ep542/README.md:34]: Added `--depth=1` to optimize cloning

File: sample_code/ep540/README.md (hunk 1 above)
Comment (int3l) [reply sample_code/ep540/README.md:10]: Basically this helps the reader to think and to replace the placeholder value. I do the same with all git hashes.

Comment (int3l) [reply sample_code/ep540/README.md:10]: If you want, I can replace it with actual url value.

Comment (asottile) [reply sample_code/ep540/README.md:10]: I think they're fairly permanent

Comment (int3l) [reply sample_code/ep540/README.md:10]: Well, yeah for one year or whatever is the expiration time :D But the value on the actual Sentry login page already changed.
The old one `https://s1.sentry-cdn.com/_static/dist/sentry/entrypoints-hashed/sentry-B0SOKci_bIC4DQfMeYs2G7ti9GKmSLzdPvA6bYL4UZk.css` gives me Access Denied ...
So do you want the new value?

Comment (asottile) [reply sample_code/ep540/README.md:10]: shrugs

Comment (int3l) [reply sample_code/ep540/README.md:10]: Sorry for the noisy comments/communication, but I was not sure as well.
I know that you have more important stuff to do.
Have a nice day!

**Pull request merged**